scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -175,6 +179,7 @@ def net_interfaces_config(args, nicmodel):
             cloud=args.cloud,
             nodecounter=args.nodecounter,
             nicindex=index,
+            net=get_net_for_nic(args, index),
```

Comment:
i.e. ironicnic == -1 indicates that there should be no ironic adapter so the previous behavior would be to put all nics on admin network.